.nxreleaserc.json

```
@@ -0,0 +1,7 @@
+{
+  "changelog": true,
+  "npm": true,
```

```suggestion
  "npm": false,
```

Como não vamos publicar pacotes no npm, podemos deixar como `false` aqui

---

Atualizei o pull request, com essa alteração. Deve ter ficado algum lixo na hora de fazer o merge.

---

Agora tá perfeito 💯